addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -498,11 +514,23 @@ var MassMailingFieldHtml = FieldHtml.extend({
             $themeSelector.find('.dropdown-item:eq(' + themesParams.indexOf(selectedTheme) + ')').addClass('selected');
         };
 
-        $themeSelector.on("click", ".dropdown-item", selectTheme);
-        $themeSelectorNew.on("click", ".dropdown-item", async (e) => {
+        $themeSelector.on('click', '.dropdown-item', (e) => {
             e.preventDefault();
             e.stopImmediatePropagation();
             const themeParams = themesParams[$(e.currentTarget).index()];
+            selectTheme(themeParams);
+        });
+        $themeSelectorNew.on('click', '.dropdown-item', async (e) => {
+            e.preventDefault();
+            e.stopImmediatePropagation();
+
+            const themeName = $(e.currentTarget).attr('id');
+
+            let themeParams = themesParams.find(theme => theme.name === themeName);
```

Comment:
Just a question of style but here it is in one line :
```js
const themeParams = [...themesParams, ...templatesParams].find(theme => theme.name === themeName);
```

Reply:
I always forget to use the `...` trick, thanks :)